.circleci/config.yml

```diff
@@ -47,8 +48,42 @@ commands:
             # "!" negates return code. exit nonzero if any of these deps are found
             ! pip freeze | grep -E "xgboost|catboost|lightgbm|plotly|ipywidgets"
             exit $?
+  build_pkg:
+    steps:
+      - run: |
+            git clone https://github.com/conda-forge/evalml-core-feedstock
+            source test_python/bin/activate
+            mkdir evalml-core-feedstock/evalml
+            cp -r `ls -A | grep -v "evalml-core-feedstock"` ./evalml-core-feedstock/evalml/
+            python conda_config.py "$(python setup.py --version)"
+            cd evalml-core-feedstock
+            echo "$DOCKER_HUB_PASS" | docker login -u psalter --password-stdin
+            export DOCKER_CONTAINERID="$(docker run -td condaforge/linux-anvil-comp7)"
+            echo "Created container ${DOCKER_CONTAINERID}"
+            chmod -R 777 ./
+            docker cp . ${DOCKER_CONTAINERID}:/home/conda/feedstock_root/
+            docker cp ./recipe/. ${DOCKER_CONTAINERID}:/home/conda/recipe_root/
+            echo "COMMITING UPDATED IMAGE"
+            docker commit ${DOCKER_CONTAINERID} psalter/build:latest
+            docker stop ${DOCKER_CONTAINERID}
+            export CONFIG=linux_64_
+            export UPLOAD_PACKAGES=False
+            export HOST_USER_ID=$(id -u)
+            export FEEDSTOCK_NAME=evalml-core-feedstock
+            docker run -t -e CONFIG -e HOST_USER_ID -e UPLOAD_PACKAGES -e GIT_BRANCH -e UPLOAD_ON_BRANCH -e CI -e FEEDSTOCK_NAME -e CPU_COUNT -e BINSTAR_TOKEN -e FEEDSTOCK_TOKEN -e STAGING_BINSTAR_TOKEN psalter/build:latest bash /home/conda/feedstock_root/.scripts/build_steps.sh
```

Comment:
If I'm following right, this command will have a non-zero return code if there's an error? Meaning the CI job will get flagged as failed?

Will the error output appear in the CI job, or do we have to do something else to get the error output? It would be amazing to have the output appear in the CI job, because otherwise its gonna be hard to debug.

Reply:
The error appears in the CI job!